src/Raven.Client/Documents/Session/ISessionDocumentCounters.cs

```diff
@@ -9,24 +9,21 @@
 namespace Raven.Client.Documents.Session
 {
     /// <summary>
-    ///     Counters advanced synchronous session operations
+    /// Provides synchronous client API for counter operations on a specific entity
```

Comment:
It is important to underline it's 'synchronous'?

Reply:
no :)
removed this